Financial Times.js

```diff
@@ -42,105 +36,94 @@
 */
 
 
-function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
-function doWeb(doc, url) { return FW.doWeb(doc, url); }
+function detectWeb(doc, url) {
+	if (url.includes('/content/')) {
+		return "newspaperArticle";
+	}
+	else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+	return false;
+}
 
- 
-/** Blog */
-FW.Scraper({
-itemType         : 'blogPost',
-detect           : FW.Xpath('//h2[@class="entry-title"]'),
-title            : FW.Xpath('//h2[@class="entry-title"]').text().trim(),
-attachments      : [
-  {
-  url: FW.Url(),
-  title: "Financial Times Snapshot",
-  type: "text/html"}],
-creators         : FW.Xpath('//span[@class="author_byline"]/a').text().replace(/^\s*by\s*/, "").cleanAuthor("author"),
-date             : FW.Xpath('//span[contains(@class, "entry-date")]').text(),
-ISSN			 : "0307-1766",
-publicationTitle : "Financial Times"
-}); 
- 
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('.search-item a.js-teaser-heading-link[href*="/content/"]');
```

Comment:
Can we omit that class? Seems pretty fragile.

Reply:
Probably, yes. I'll give it a try.